docs/contributing/copyright-license/index.md

```diff
@@ -10,7 +10,19 @@ sidebar_position: 10
 
 ## License
 
-Most of Snowplow components are licensed either under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0), or the [Snowplow Community License](/community-license-1.0/). Examples for the latter include [Snowbridge](https://github.com/snowplow/snowbridge) and [RDB Loader](https://github.com/snowplow/snowplow-rdb-loader) (since version 5.7.0).
+In general, Snowplow components are licensed under one of the following 3 licenses:
+
+* [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0) for trackers and various SDKs.
+* [Snowplow Community License](/community-license-1.0/) for core pipeline components, e.g. [RDB Loader](https://github.com/snowplow/snowplow-rdb-loader) or [Snowbridge](https://github.com/snowplow/snowbridge), as well as Terraform modules to run them. ([FAQ](/docs/contributing/community-license-faq/index.md))
+* [Snowplow Personal & Academic License](/personal-and-academic-license-1.0/) for data models. ([FAQ](/docs/contributing/personal-and-academic-license-faq/index.md))
```

Comment:
My understanding, which may be wrong, is that the "core" data models will remain under the SCL, but new and specialised ones will migrate to the SPAL?

Reply:
@stanch I would change "for data models" to "for selected data models and other commercial components"

Reply:
I believe it’s correct as stated